src/main/java/com/iexec/worker/tee/scone/SconeLasConfiguration.java

```diff
@@ -33,8 +29,14 @@
 
     private String containerName;
 
-    public void setContainerName(String containerName) {
-        this.containerName = containerName;
+    public SconeLasConfiguration(WorkerConfigurationService workerConfigService) {
+        // We use "worker-name-iexec-las" as containerName to avoid conflict
```

Comment:
Great idea, should we do ?
worker-name-0x1wallet23456789-iexec-las
(I've seen that in another PR)

Reply:
The other PR was `worker-name-0xchaintaskid...` for the container of the task, but I like the idea. I'll add the worker wallet address.